File: nodejs/fib.js
const fibonacci = (n) => {
  return (n <= 1) ? n : (fibonacci(n - 1) + fibonacci(n - 2));
};

try {
  const sequences = Number(process.argv[2]);
  const start = process.hrtime();
  const output = fibonacci(sequences);
  const time = process.hrtime(start);
  const elapsed = time[0] * 1000 + time[1] / 1000000;
  console.log(`Node.js Fib: ${sequences}x\nResult: ${output} | Time: ${elapsed.toFixed(0)}ms`);
} catch (e) {
  console.log(e);
}

File: golang/fib.go
package main

import (
	"fmt"
	"time"
	"strconv"
	"os"
)

func Fibonacci(n uint64) uint64 {
	if n <= 1 {
		return n
	}
	return Fibonacci(n - 1) + Fibonacci(n - 2)
}

func main() {
	fmt.Print("Golang Recursive Fib Benchmark...\n")
	sequences, err := strconv.ParseUint(os.Args[1], 10, 64)
	if err != nil {
		fmt.Println("Invalid argument")
	} else {
		fmt.Printf("Calculating fib %dx...\n", sequences)
		start := time.Now()
		output := Fibonacci(sequences)
		elapsed := time.Since(start)
		fmt.Printf("Result: %d\nTime elapsed: %.3fms\n", output, (elapsed.Seconds() * 1000))
	}
}